### ISSUE TITLE
⌨ Mobile menu focus order issue

### ISSUE THREAD

**Issue by BrookeCameron** [GCTC-NTGC/gc-digital-talent#15925]
#### 🛡️Fable user reported: 
When the menu button is expanded all the menu options appear before the collapseable/expandable button. The options should appear after the menu button in the swipe order. 

#### Expected behaviour: 
Collapse/expand should be the first option, you shouldn't have to navigate through every other choice before being able to close the menu.

### WCAG 2.1 Criteria

2.4.3 Focus Order

### 📋 Steps to Reproduce

1. Sign in 
2. expand the menu and navigate to collapse

### What operating systems are you encountering the problem on?

iOS

### What browsers are you encountering the problem on?

Firefox

### What assistive technologies are you encountering the problem on?

VoiceOver

### ✅ Acceptance Criteria

- [ ] Collapse/expand is the first choice in the focus order.

**Comment by petertgiles** [comment]
Reviewed refinement:
- Because the trigger is at the bottom the reading order is backwards between mobile and desktop views
- Solving this would probably require separating mobile and desktop components or designing a top-to-bottom menu

**Comment by mnigh** [comment]
@yonikid15 are you working on this issue since you have been assigned it?
